developer_tools/XSDSchemas/Optimizers.xsd

```diff
@@ -196,6 +196,7 @@
      <xsd:complexType name="mutationType">
       <xsd:all>
         <xsd:element name="mutationProb"   type="xsd:float" />
+        <xsd:element name="locs"           type="xsd:string" minOccurs="0" maxOccurs="1"/>
```

Comment:
Is this new tag? If so, please update user manual?

Reply:
The manual is up to date, we realized we forgot to update the XSD when we put the new tests. Ref. line 1089 of optmizer.tex